js-api/src/dapi.ts

```diff
@@ -381,6 +381,43 @@ export class AdminDataSource {
   getServiceInfos(): Promise<ServiceInfo[]> {
     return new Promise((resolve, reject) => api.grok_Dapi_Admin_GetServiceInfos(this.dart, (q: any) => resolve(toJs(q)), (e: any) => reject(e)));
   }
+
+
+  /**
+   * Sends envelope
+   * @param envelope - message that will be sent using configured SMTP service
+   */
+  sendEmail(envelope: Envelope): Promise<void> {
+    if (envelope.to.length === 0)
+      throw new Error('Recipients list shouldn\'t be empty');
+    return api.grok_Dapi_Admin_Send_Email(this.dart, envelope);
+  }
+}
+
+/**
+ * Represents message that can be sent over the network using the configured SMTP service
+ */
+export interface Envelope {
```

Comment:
Shouldn't we call it Email, or is Envelope a commonly used name in such cases?

Reply:
Yeah, I read a little about the semantics of these words, and an envelope represents a message that can be sent to only one recipient. In our case it is incorrect. I will change it to `Email`;